molo/forms/tests/test_reaction_questions.py

```diff
@@ -0,0 +1,528 @@
+            content_type=wagtailadmin_content_type,
+            codename='access_admin',
+            name='Can access Wagtail admin'
+        )
+        # Create reaction question view response
+        self.reaction_question = Permission.objects.get(
+            content_type=reaction_question_content_type,
+            codename='can_view_response')
+        # create a group
+        self.test_group, _ = Group.objects.get_or_create(name='Test group')
+        self.test_group.permissions.add(access_admin)
+        # create a user and add the user to the group
+        user = User.objects.create_user(
+            username='username',
+            password='password',
+            email='login@email.com',
+            is_staff=True,
+        )
+        user.groups.add(self.test_group)
+
+        self.reaction_index = self.get_reaction_index(self.main)
+
+    def test_superuser_can_see_reaction_question_modeladmin(self):
+        User.objects.create_superuser(
+            username='super', password='password', email='login@email.com')
+        self.client.login(username='super', password='password')
+
+        response = self.client.get('/admin/')
+        self.assertEqual(response.status_code, 200)
+        self.assertContains(response, 'reactionquestion')
+
+    def test_user_has_perm_can_see_reaction_question_modeladmin(self):
+        self.client.login(username='username', password='password')
+        user = User.objects.filter(username='username').first()
+
+        # User shoudn't see the reaction question model admin
+        self.assertTrue(user.has_perm('wagtailadmin.access_admin'))
+        self.assertFalse(user.has_perm('forms.can_view_response'))
+
+        response = self.client.get('/admin/')
+        self.assertEqual(response.status_code, 200)
+        self.assertNotContains(response, 'reactionquestion')
+
+        # User shoud see the reaction question model admin
+        self.test_group.permissions.add(self.reaction_question)
+        user = User.objects.filter(username='username').first()
+        self.assertIn(self.test_group, user.groups.all())
+
+        self.assertTrue(user.has_perm('wagtailadmin.access_admin'))
+        self.assertTrue(user.has_perm('forms.can_view_response'))
+
+        response = self.client.get('/admin/')
+        self.assertEqual(response.status_code, 200)
+        self.assertContains(response, 'reactionquestion')
+
+
+class TestReactionQuestionChoiceFeedbackView(FormsTestCase, TestCase):
+    """Test response reaction """
+
+    def setUp(self):
+        self.mk_main()
+        self.english = SiteLanguageRelation.objects.create(
+            language_setting=Languages.for_site(self.main.get_site()),
+            locale='en',
+            is_active=True)
+
+        self.assertTrue(self.section_index)
+        self.yourmind = self.mk_section(
+            self.section_index, title='Your mind')
+        self.reaction_index = self.get_reaction_index(self.main)
+
+    def test_view(self):
```

Comment:
Please add a more detailed name or comment here. What is the view that you're testing expected to do?